tools/bag_processing/scripts/rosbag_fix_all.py

```diff
@@ -174,15 +174,15 @@ class CustomFormatter(argparse.ArgumentDefaultsHelpFormatter):
     )
     parser.add_argument(
         "--decode-haz",
-        default="",
         help="decode the extended haz cam topic into points and amplitude_int, argument is robot name",
+        default="",
         type=str,
     )
     parser.add_argument(
         "--filter",
-        help="filter the bagfile with only some topics",
+        help='filter the bagfile. Use with "" quotes. Example:  --filter "--accepts /hw/imu --regects /loc/ml/features"',
```

Comment:
Should be `--accept` and `--reject`

Reply:
Hm, separate thought... I guess to make it more realistic, it could be `"--accept '/loc/*' --reject '/loc/ml/features'"`. Suggesting the change because, in your prior example, the `--reject` argument would have no effect, so it might tend to confuse people about the filtering semantics.

Reply:
thanks for catching that!